docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -289,3 +289,15 @@ Property Name                                    Description
 ``elasticsearch.aws.secret-key``                 AWS secret key to use to connect to the Elasticsearch domain.
 ================================================ ==================================================================
 
+Password Authentication
+-----------------------
+
+To enable password authentication, the ``elasticsearch.security`` option needs to be set to ``PASSWORD``.
```

Comment:
We could, but I like that the sections are self-contained right now. I think we should get this in as is since we don't have any docs about password auth, and improve later if we so desire.